From 2a36a7f24e71e55df5ce470276ebb1c06416e585 Mon Sep 17 00:00:00 2001
From: Adam YH Lee <adam.yh.lee@gmail.com>
Date: Wed, 24 Jun 2015 13:41:17 -0700
Subject: [PATCH 2/3] Update rootfs type to ext4 for DuoVero

Gumstix is migrating the file system type to ext4. Unlike Overo and
Pepper, DuoVero does not have its own U-Boot environment variable. It
uses the default values defined in the ti_omap4_common.h file.
---
 include/configs/ti_omap4_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/ti_omap4_common.h b/include/configs/ti_omap4_common.h
index e966134..f2673fc 100644
--- a/include/configs/ti_omap4_common.h
+++ b/include/configs/ti_omap4_common.h
@@ -92,7 +92,7 @@
 	"vram=16M\0" \
 	"mmcdev=0\0" \
 	"mmcroot=/dev/mmcblk0p2 rw\0" \
-	"mmcrootfstype=ext3 rootwait\0" \
+	"mmcrootfstype=ext4 rootwait\0" \
 	"mmcargs=setenv bootargs console=${console} " \
 		"vram=${vram} " \
 		"root=${mmcroot} " \
-- 
2.1.4

